mumd/src/audio.rs

```diff
@@ -173,7 +180,7 @@ impl Audio {
             opus::Application::Voip,
         )
         .unwrap();
-        let (input_sender, input_receiver) = mpsc::channel(100);
+        let (input_sender, _) = broadcast::channel(100);
```

Comment:
Looks weird ignoring the receiver. Should probably rework so that the receiver is used as well.